pyzo/pyzokernel/interpreter.py

```diff
@@ -1172,34 +1172,28 @@ def showtraceback(self, useLastTraceback=False):
         try:
             if useLastTraceback:
                 # Get traceback info from buffered
-                type = sys.last_type
+                etype = sys.last_type
                 value = sys.last_value
                 tb = sys.last_traceback
             else:
                 # Get exception information and remove first, since that's us
-                type, value, tb = sys.exc_info()
+                etype, value, tb = sys.exc_info()
                 tb = tb.tb_next
 
                 # Store for debugging, but only store if not in debug mode
                 if not self._dbFrames:
-                    sys.last_type = type
+                    sys.last_type = etype
                     sys.last_value = value
                     sys.last_traceback = tb
 
-            lines = traceback.format_exception(value)
-
-            # Remove exec() call in interpreter.py"
-            for i in range(max(len(lines), 5)):
-                if "pyzo" in lines[i] and "interpreter.py" in lines[i] and "exec(" in lines[i]:
-                    lines.pop(i)
-                    break
```

Comment:
This is completely removed, but that means the line showing Pyzo internals will be shown in the tracebacks, I think?

Reply:
No. As in older Pyzo versions, Pyzo's stack frame is already removed a few lines before that:
https://github.com/pyzo/pyzo/blob/b46302d5f14d2771e09336847ed4b252432fe3df/pyzo/pyzokernel/interpreter.py#L1173-L1175

With this new fix, the modified traceback object `tb` is passed to `format_exception`:
`lines = traceback.format_exception(etype, value, tb)`
We need to pass it anyways, because only Python >= v3.10 allows single argument calls of `traceback.format_exception`.

In your previous fix, you did not pass the modified traceback `tb`, so `format_exception` internally used the unmodified traceback:
`lines = traceback.format_exception(value)`